src/app/directives/resizable.directive.ts

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2020-2021 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import { Directive, ElementRef, OnInit, Input, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[mdmResizable]' // Attribute selector
+})
+
+export class ResizableDirective implements OnInit {
+
+
+  @Input() resizableGrabWidth = 4;
+  @Input() resizableMinWidth = 10;
+
+  dragging = false;
+
+  constructor(private el: ElementRef, private renderer: Renderer2) {
+
+
+
+    const newWidth = (wid) => {
+      const newWidthCalc = Math.max(this.resizableMinWidth, wid);
+      this.renderer.setStyle(this.el.nativeElement,'width',`${newWidthCalc}px`);
+    };
+
+
+    const mouseMoveG = (evt) => {
```

Comment:
Does the "G" in these names refer to "Global"? If so could we rename them to `mouseMoveGlobal()` etc to make them clearer?